functions/middleware/httperror.js

```diff
@@ -0,0 +1,16 @@
+const logger = require('./logger');
+
+module.exports = function(err, request, response, next) {
+    err.statusCode = err.statusCode || 500;
+    if(err.statusCode === 500) {
+       err.status =  'Internal Server Error'
+       logger.error(err.stack);
+    } else {
+        err.status = err.message || 'Internal Server Error';
+    }
+    logger.error(err)
+    response.status(err.statusCode).json({
+        status: err.statusCode,
```

Comment:
please remove this line, as it is redundant.

Reply:
I have removed it. It now returns
{
"message": "Requested product sdfsd is not present in Firestore."
}

without status. code

Reply:
Done.